api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -511,6 +558,64 @@ public void validate_OperationalUser_nullInstitution() {
     assertThat(service.validateOperationalUser(institution)).isFalse();
   }
 
+  @Test(expected = BadRequestException.class)
```

Comment:
I like these.  Could you add one more for enforcing that OtherText exists when the type is OTHER?